test/features/deploy_kogito_build_vs_code.feature

```diff
@@ -0,0 +1,31 @@
+Feature: Deploy Kogito Build - after assets processed by VS code
+  
+
+  Background:
+    Given Namespace is created
+    And Kogito Operator is deployed
+    And Project kie-asset-library is cloned
+    And Project kie-asset-library is built by maven with configuration:
+      | profile | dmn                                    |
+      | option  | -Dactive.structures=quarkus            |
+      | option  | -Dactive.config.sets=scesim            |
+      | native  | disabled                               |
+    And Project "<project>" is generated by kie-asset-library
+
+
+  Scenario Outline: Build <runtime> binary build with native <native> using KogitoBuild
+    Given Project "<project>" assets are re-marshalled by VS Code
+    And Project "<project>" in kie-asset-library is built by maven
+      
+    When Deploy <runtime> project "<project>" from kie-asset-library on openshift with configuration:
```

Comment:
Please drop openshift mention, the tests could be executed on plain Kubernetes too.

Reply:
good point, addressed in last commit